{"name": "Full End-to-End Booking Flow", "status": "broken", "statusDetails": {"message": "KeyError: 'label_type'", "trace": "self = <allure_pytest.listener.AllureListener object at 0x0000011940FEB8C0>, item = <Function test_run_full>\n\n    @pytest.hookimpl(hookwrapper=True)\n    def pytest_runtest_teardown(self, item):\n        yield\n        uuid = self._cache.get(item.nodeid)\n        test_result = self.allure_logger.get_test(uuid)\n        test_result.historyId = get_history_id(\n            test_result.fullName,\n            test_result.parameters,\n            original_values=self.__get_pytest_params(item)\n        )\n>       test_result.labels.extend([Label(name=name, value=value) for name, value in allure_labels(item)])\n\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_pytest\\listener.py:145: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nitem = <Function test_run_full>\n\n    def allure_labels(item):\n        unique_labels = dict()\n        labels = set()\n        for mark in item.iter_markers(name=ALLURE_LABEL_MARK):\n>           label_type = mark.kwargs[\"label_type\"]\nE           KeyError: 'label_type'\n\n..\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_pytest\\utils.py:61: KeyError"}, "attachments": [{"name": "log", "source": "819f03c9-efbf-4f72-b51c-adf57d38c090-attachment.txt", "type": "text/plain"}], "start": 1745324635826, "stop": 1745324715059, "uuid": "84351512-d9ea-4c56-abfd-84b309792f1f", "historyId": "12c417b5a7a6bf8402323a2990001b5f", "testCaseId": "12c417b5a7a6bf8402323a2990001b5f", "fullName": "tests.ui.test_run_full#test_run_full"}